src/v/rpc/transport.h

```diff
@@ -159,13 +163,32 @@ template<typename Input, typename Output>
 inline ss::future<result<client_context<Output>>>
 transport::send_typed(Input r, uint32_t method_id, rpc::client_opts opts) {
     using ret_t = result<client_context<Output>>;
+    return send_typed_versioned<Input, Output>(
+             std::move(r), method_id, std::move(opts), transport_version::v0)
+      .then([](result<result_context<Output>> r) {
```

Comment:
nit: it's obvious upon reading, but consider using a name other than 'r' to avoid confusion with the Input variable. Same elsewhere (though if this is widely entrenched in existing code, feel free to ignore)

Reply:
yeh, it's muscle memory from pre-coroutines times where 80 column width limit was exhausted very quickly with nested continuations.

Reply:
renamed